ideas/167-extended-automated-testing-for-continuous-delivery/README.md

```diff
@@ -0,0 +1,58 @@
+## Preamble
+
+    Idea: 167
+    Title: Functional automated testing as a part of Continuous Delivery process
+    Status: Draft
+    Created: 2018-04-12
+
+## Summary
+For automated testing to be a part of CD process we need to:
+- increase coverage
+- increase reliability
+- provide visible results on daily basis
+
+## Swarm Participants
+- Lead Contributor: @antdanchenko
+- Testing & Evaluation: @antdanchenko
+- Contributor: @yevh-berdnyk
+- Contributor: @lukaszfryc
+- PM:
+
+## Product Overview
+Extended set of automated tests which will run against each nightly build in order to provide results via TestRun in TestRail + short summary via Slack notification
```

Comment:
TestRail is something that only testers use. But, it's needed to have the history. On the other hand, I like the Slack notification. The notification could compare current results with previous ones.

Reply:
We can share TestRail with any team member
Notifications is just a summary with number of passed and failed tests